doc/api/crypto.markdown

```diff
@@ -605,6 +605,14 @@ encoding of `'binary'` is enforced. If `data` is a [`Buffer`][] then
 
 This can be called many times with new data as it is streamed.
 
+### hmac.validate(inputHmac)
```

Comment:
This might be a little misleading since `hmac.validate(input)` computes the HMAC of `input` and does not take in an already computed `inputHMAC`.
So maybe change the description to
### hmac.validate(input)
```md
Return true if and only if the computed hmac matches the hmac of the
provided input. The input argument must be provided as a Node.js Buffer.
```
How does that look?